extensions/amp-accordion/1.0/amp-accordion.css

```diff
@@ -29,3 +33,25 @@
   padding-right: 20px;
   border: solid 1px #dfdfdf;
 }
+
+/* TODO: This CSS is duplicated from ./accordion.jss.js for purposes of display
+ * locking.  Look into de-duping once experiment is launched.
+ */
+amp-accordion.i-amphtml-display-locking
+  > section:not([expanded])
+  > :last-child:not(.i-amphtml-animating),
+amp-accordion.i-amphtml-display-locking
+  > section:not([expanded])
+  > :last-child:not(.i-amphtml-animating)
+  * {
+  content-visibility: hidden-matchable !important;
+  display: block !important;
+}
+
+amp-accordion > section:not([expanded]) > :last-child:not(.i-amphtml-animating),
```

Comment:
Please bring these ones above the experimental version.

Reply:
Updated in latest revision.